src/pages/AzurePreview.js

```diff
@@ -1,89 +1,235 @@
-import { div, h, h1, p } from 'react-hyperscript-helpers'
-import { ButtonOutline, ButtonPrimary, Link } from 'src/components/common'
+import { Fragment, useCallback, useEffect, useState } from 'react'
+import { div, form, h, h1, input, p } from 'react-hyperscript-helpers'
+import { ButtonOutline, ButtonPrimary } from 'src/components/common'
+import { icon } from 'src/components/icons'
+import { ValidatedInput } from 'src/components/input'
 import planet from 'src/images/register-planet.svg'
 import { ReactComponent as TerraOnAzureLogo } from 'src/images/terra-ms-logo.svg'
+import { Ajax } from 'src/libs/ajax'
 import { signOut } from 'src/libs/auth'
 import colors from 'src/libs/colors'
+import { reportError } from 'src/libs/error'
+import { FormLabel } from 'src/libs/forms'
+import { getLocalPref, setLocalPref } from 'src/libs/prefs'
 import { useStore } from 'src/libs/react-utils'
-import { authStore, azurePreviewStore } from 'src/libs/state'
-import * as Utils from 'src/libs/utils'
+import { authStore, azurePreviewStore, getUser } from 'src/libs/state'
 
 
-const AzurePreview = () => {
-  // State
-  const { isAzurePreviewUser } = useStore(authStore)
+const styles = {
+  centered: {
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+  },
+  paragraph: {
+    fontSize: 16,
+    lineHeight: 1.5,
+    maxWidth: 570,
+  },
+  header: {
+    display: 'flex',
+    marginTop: '3rem',
+    marginBotton: '2rem',
+    color: colors.dark(0.8),
+    fontSize: '1.8rem',
+    fontWeight: 500,
+  },
+  button: {
+    textTransform: 'none',
+  },
+}
 
-  // Helpers
-  const styles = {
-    centered: {
-      display: 'flex',
-      justifyContent: 'center',
-      alignItems: 'center'
+const AzurePreviewForPreviewUser = () => {
+  const dismiss = () => {
+    azurePreviewStore.set(true)
+  }
+
+  return h(Fragment, [
+    p({ style: styles.paragraph }, [
+      'This is a preview version of the Terra platform on Microsoft Azure.'
```

Comment:
I notice the "allowed user" text is not centered anymore.

Before:

<img width="598" alt="image" src="https://user-images.githubusercontent.com/5368863/214198871-147307e2-f45d-463b-9aa6-09c065b2dc65.png">


After:

<img width="621" alt="image" src="https://user-images.githubusercontent.com/5368863/214198903-4f9f949f-10d8-454b-bc46-05df3a103978.png">


I kind of like the centered version, but am OK if this was intentional.

Reply:
The InVision design for the form and confirmation had everything left aligned. I followed that for the splash screen shown to preview users as well.

Reply:
There's a request to add another field, so changing the alignment is low on my priority list.

I thought @lstrano-broad made the mockups with the left alignment?